charts/kube-otel-stack/values.yaml

```diff
@@ -65,19 +83,53 @@ tracesCollector:
           name: otel-collector-secret
   config:
     receivers:
+      # OTel-Arrow notes: For this collector to receive OTel-Arrow in
+      # addition to standard forms of OTLP, use an image with the
+      # OTel-Arrow receiver component replace "otlp" below with
+      # "otelarrow".
       otlp:
         protocols:
           grpc:
             endpoint: "0.0.0.0:4317"
+          http:
+            endpoint: "0.0.0.0:4318"
```

Comment:
Incidentally, I think we should recommend opening the default HTTP port, e.g.,

Reply:
@jmacd can we do the same in the otel-cloud-stack chart as well? I agree good thought